code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -68,6 +68,8 @@
 			return pick('sound/voice/human/malescream_1.ogg', 'sound/voice/human/malescream_2.ogg', 'sound/voice/human/malescream_3.ogg', 'sound/voice/human/malescream_4.ogg', 'sound/voice/human/malescream_5.ogg', 'sound/voice/human/malescream_6.ogg')
 	else if(ismoth(H))
 		return 'sound/voice/moth/scream_moth.ogg'
+	else if(islizard(H))
+		return pick('sound/voice/lizard/lizard_scream_1.ogg', 'sound/voice/lizard/lizard_scream_2.ogg', 'sound/voice/lizard/lizard_scream_3.ogg', 'sound/voice/lizard/lizard_scream_4.ogg')
```

Comment:
You need to make it clear in the code that it's a Fulp change, if it's impossible for you to make the change into something modular. After looking at it a little closer, it does seem pretty hard to do, so that's why you need to make sure that it's very, very explicit in the code that it's a Fulp-only change. I'll suggest a change so you see how it's done.

Reply:
```suggestion
    // [FULP EDIT STARTS] - Adding lizard screams
	else if(islizard(H))
		return pick('sound/voice/lizard/lizard_scream_1.ogg', 'sound/voice/lizard/lizard_scream_2.ogg', 'sound/voice/lizard/lizard_scream_3.ogg', 'sound/voice/lizard/lizard_scream_4.ogg')
		// [FULP EDIT END]
```
This should be a better way to do it.

Reply:
Couldn't this be made 100% modular by copying the datum on a fulp_modules file, with a . = ..() check at the beginning, and THEN add the lizard check?